src/course-unit/course-xblock/CourseXblock.jsx

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import { Dropdown, Icon, IconButton } from '@edx/paragon';
+import { EditOutline as EditIcon, MoreVert as MoveVertIcon } from '@edx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import messages from './messages';
+
+const CourseXblock = ({ id, title }) => {
+  const intl = useIntl();
+
+  return (
+    <li className="studio-xblock-wrapper mb-4">
```

Comment:
[question] Why did you decide to stop using [the Card from the Paragon component](https://paragon-openedx.netlify.app/components/card/#horizontal-variant-1)?

Reply:
Changed, thanks. We need to add paragon component's name to the ticket description for the future.